### PR TITLE
Introduce the `lmdb_version` function

### LINKED ISSUE #199: Expose a function to return the LMDB version
I want to expose a very simple function that returns the major, minor, and patch versions of the underlying LMDB library.

```rust
fn heed::lmdb_version() -> (major, minor, patch);
```

### PR DESCRIPTION
**Pull request by postmeback**
# Pull Request

## Related issue
Fixes #199 

## What does this PR do?
-  Adds a function in lib.rs, to expose the LMDB version.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: heed/src/lib.rs [hunk 1]
```diff
@@ -140,6 +140,17 @@ impl From<io::Error> for Error {
...
 
+pub fn lmdb_version() -> (i32, i32, i32) {
+    let mut version = 0;
+    unsafe {
+        mdb::ffi::mdb_version(&mut version, std::ptr::null_mut(), std::ptr::null_mut());
+    }
+    let major = version >> 16;
+    let minor = (version >> 8) & 0xFF;
+    let patch = version & 0xFF;
+    (major as i32, minor as i32, patch as i32)
```
Comment (Kerollmops) [inline heed/src/lib.rs:151]: [According to the documentation](https://github.com/meilisearch/heed/blob/e78600d32634448375d8b2661c96366383f4d93a/lmdb-master-sys/src/bindings.rs#L175-L180), the three numbers are important.

Can I know where you found the idea of doing all those bit operations, please? Were you getting help from a tool or something?

```suggestion
    let mut major = mem::MaybeUninit::uninit();
    let mut minor = mem::MaybeUninit::uninit();
    let mut patch = mem::MaybeUninit::uninit();
    unsafe {
        mdb::ffi::mdb_version(major.as_mut_ptr(), minor.as_mut_ptr(), patch.as_mut_ptr());
        (major.assume_init(), minor.assume_init(), patch.assume_init())
    }
```

**Review by Kerollmops — CHANGES_REQUESTED** [review]

File: heed/src/lib.rs (hunk 1 above)
Comment (postmeback) [reply heed/src/lib.rs:151]: Yes, I got help from chatgpt.

**Comment by postmeback** [comment]
Committed with your suggestion.

**Comment by Kerollmops** [comment]
Thank you!

Would it be possible to make the CI pass, add documentation and a documentation example, please?

**Comment by postmeback** [comment]
Can you give me an example, of where to write the documentation and example? @Kerollmops

**Pull request closed without merging**